frontend/src/App.tsx

```diff
@@ -85,9 +86,14 @@ const Scene = () => {
 
 const App = () => {
   return (
-    <Canvas shadows dpr={[1, 2]} camera={{ position: [0, 0, 10], fov: 50 }}>
-      <Scene />
-    </Canvas>
+    <>
+      <React.StrictMode>
+        <Canvas shadows dpr={[1, 2]} camera={{ position: [0, 0, 10], fov: 50 }}>
+          <Scene />
+        </Canvas>
+      </React.StrictMode>
+      <SpotifyPlayer></SpotifyPlayer>
```

Comment:
NIT this can be `<SpotifyPlayer />` right?

Reply:
yep